softdev.bib

```diff
@@ -2479,3 +2479,25 @@ @inproceedings{rohou15branch
     month = Feb,
     doi= "10.1109/CGO.2015.7054191"
 }
+
+@inproceedings{prokopec19renaissance,
```

Comment:
Can we use the accents directly rather than the old TeX stuff. All our papers expect UTF-8 input these days.

Reply:
Yep!